### PR TITLE
[miio] add support Xiaomi Smart Air Purifier 4 (modelId: zhimi.airp.mb5)

### LINKED ISSUE #12818: [miio] Add support for Xiaomi Mi Air Purifier 4 zhimi.airp.mb5
Hello,
I tried adding support myself with the [(experimental) create experimental support for new miot protocol device]. It generated the JSON. However, the automatic update of the thing did not work. With some name changing, creating a new thing, and changing binding setting i was able to get the JSON to be accepted and used. I could not test all channels, but they should be similar to the existing Air Purifiers. I tested the ion switch which is new to the Air Purifier 4.

Here is the JSON as txt.
[zhimi.airp.mb5.txt](https://github.com/openhab/openhab-addons/files/8781023/zhimi.airp.mb5.txt)

I hope this is everything required to add the device.

Best regards

### PR DESCRIPTION
**Pull request by marcelrv**
Adding support for the following models:
* Xiaomi Smart Air Purifier 4 (modelId: zhimi.airp.mb5)

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

close #12818

### CODE REVIEW

**Comment by marcelrv** [comment]
@lolodomo if someone can merge this one as well, I happily leave the top 10 :-)

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airp.mb5-miot.json [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				],
+				"readmeComment": "Value mapping `[\"1\"\u003d\"Celsius\",\"2\"\u003d\"Fahrenheit\"]`"
+			},
+			{
+				"property": "moto-speed-rpm",
```
Comment (lolodomo) [inline bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airp.mb5-miot.json:434]: motor?

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airp.mb5-miot.json [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+				"siid": 9,
+				"piid": 1,
+				"friendlyName": "Custom Service - Moto Speed Rpm",
+				"channel": "moto_speed_rpm",
```
Comment (lolodomo) [inline bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airp.mb5-miot.json:438]: motor?

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
@marcelrv : any chance you looked at my comments before Sunday?

**Comment by marcelrv** [comment]
@lolodomo Thanks, I missed your comments indeed. Will aim to look at that indeed before sunday.
I did this as bit quick one... so the values are right from the published spec.. jinglish indeed ;-)

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**